source/common/protobuf/utility.cc

```diff
@@ -118,6 +118,12 @@ void MessageUtil::checkForDeprecation(const Protobuf::Message& message, bool war
       continue;
     }
 
+    bool warn_only = true;
+    if (runtime && !runtime->snapshot().deprecatedFeatureEnabled(
```

Comment:
I assume we are checking for runtime existence because we might deprecate something before runtime is available? Worth a comment?